windows/getopt.h

```diff
@@ -1,4 +1,10 @@
 #ifndef __GETOPT_H__
+
+#include "simdjson.h" // For the warning disable
```

Comment:
getopt to include simdjson?.. that's weird.
In general, two things are there for the PR: all these deprecate warnings in simdjson.h/cpp: these are handled OK now. For all the tests, just define _CRT_SECURE_NO_DEPRECATE (or define it for everything, but not in simdjson.h)

Reply:
Yeah, it's totally weird. I'll add _CRT_SECURE_NO_DEPRECATE for tests and tools.